react/features/analytics/handlers/amplitude/fixDeviceID.native.ts

```diff
@@ -18,6 +21,12 @@ export async function fixDeviceID(amplitude: Amplitude) {
     } else {
         const uid = DeviceInfo.getUniqueId();
 
+        if (!uid) {
```

Comment:
The API has changed, watch out.

Reply:
Updated.

Reply:
No, it returns a promise: https://github.com/react-native-device-info/react-native-device-info#getuniqueid you want to use the sync version now.